.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 11.15.0
+yarn 1.22.10
```

Comment:
Could you move this file to your home directory and `git rm` this file to remove it from the commit?